cmd/sql/flow.go

```diff
@@ -264,6 +315,21 @@ func initCommand() *cobra.Command {
 	return cmd
 }
 
+func configCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "config",
+		Args:         cobra.MaximumNArgs(1),
+		RunE:         executeConfig,
+		SilenceUsage: true,
+	}
+	cmd.SetHelpFunc(executeHelp)
+	cmd.Flags().StringVar(&projectDir, "project-dir", ".", "")
+	cmd.Flags().StringVar(&environment, "env", "default", "")
+	cmd.Flags().StringVar(&key, "key", "", "")
+	cmd.MarkFlagRequired("key") //nolint:errcheck
```

Comment:
We should ideally let this through and let sql_cli handle it.
```suggestion
```

Reply:
yes, makes sense. Previously we had it as a flag, but have converted it to a required argument now in commit https://github.com/astronomer/astro-sdk/pull/1338/commits/b1d35465ced1484974c40abc076fec01096f72e3